make_deb

```diff
@@ -119,7 +119,7 @@ echo -n "Populating the package and updating version file..."
 cd "${package_name}"
 cp -R ${GIT_ROOT}/packages/Debian/${architecture}/* .
 sed -i "s/Version: 1.0.0/Version: $version/g" DEBIAN/control
-sed -i "s/Depends: foglamp/Depends: foglamp (=$foglamp_version)/g" DEBIAN/control
+sed -i "s/Depends: foglamp/Depends: foglamp (>=$foglamp_version)/g" DEBIAN/control
```

Comment:
https://github.com/foglamp/foglamp-south-sinusoid/blob/master/VERSION.south.sinusoid

foglamp_version>=1.2 

To keep it more clear for the end user.